### ISSUE TITLE
Composer error when installing the package 

### ISSUE THREAD

**Issue by Odilbukh** [noplanman/php-telegram-bot-manager#74]
### Bug Report
When I run `composer require php-telegram-bot/telegram-bot-manager:^1.6 -W` it returns this message:

Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - php-telegram-bot/telegram-bot-manager 1.6.0 requires longman/telegram-bot ^0.70 -> found longman/telegram-bot[0.70.0, 0.70.1] but it conflicts with your root composer.json require (^0.77.1).
    - php-telegram-bot/telegram-bot-manager 1.7.0 requires longman/telegram-bot ^0.73 -> found longman/telegram-bot[0.73.0, 0.73.1] but it conflicts with your root composer.json require (^0.77.1).
    - Root composer.json requires php-telegram-bot/telegram-bot-manager ^1.6 -> satisfiable by php-telegram-bot/telegram-bot-manager[1.6.0, 1.7.0].

I'm using Laravel 8 and PHP 8.0

**Comment by noplanman** [comment]
Hi @Odilbukh

To use the latest version of the core library with this manager, you need to manually define the version like this:
```json
"require": {
    "php-telegram-bot/telegram-bot-manager": "^1.7",
    "longman/telegram-bot": "0.77.1 as 0.73"
}
```

Or to use the most recent version:
```json
"require": {
    "php-telegram-bot/telegram-bot-manager": "^1.7",
    "longman/telegram-bot": "dev-master as 0.73"
}
```

**Comment by noplanman** [comment]
I've just released version 2.0.0, which defaults to the currently latest core version 0.78.0.

**Comment by Odilbukh** [comment]
Thanks!
